### PR TITLE
update formatting

### PR DESCRIPTION
**Pull request by YEOWEIHNGWHYELAB**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -783,9 +783,9 @@ _________________________________________________________
 
 # <a id="csv"></a> &nbsp;&nbsp; CSV: `csv`
 
-If our statistical analysis feature is not good enough for you, you may convert the database records
-into a csv file by simple keying in the `csv` command. You will then be able to use Excel to do advanced
-statistical analysis.
+> 😃 If our statistical analysis feature is not good enough for you, you may convert the database records
+> into a csv file by simply keying in the `csv` command. You will then be able to use Excel to do advanced
+> statistical analysis.
```
Comment (ZenoNexus) [inline docs/UserGuide.md:788]: would be better if changed to: "This feature allows you to export your data into a CSV file, for users who desire more flexibility with customizing their data and further analysis which is not yet supported by Budget Tracker."

File: docs/UserGuide.md [hunk 2]
```diff
@@ -817,9 +817,9 @@ _________________________________________________________
 
 # <a id="db"></a> &nbsp;&nbsp; DB: `db`
 
-😃 What if you have forgotten all the database years that you have been using? Fear not! `db` command 
-is here to help! `db` command basically list out all the available database text file you currently
-have. 
+> 😃 What if you have forgotten all the database years that you have been using? Fear not! `db` command 
+> is here to help! `db` command basically list out all the available database text file you currently
```
Comment (ZenoNexus) [inline docs/UserGuide.md:821]: "basically list out" can change to "lists out"

**Review by ZenoNexus — CHANGES_REQUESTED** [review]
some minor sentence improvements needed

File: docs/UserGuide.md (hunk 1 above)
Comment (YEOWEIHNGWHYELAB) [reply docs/UserGuide.md]: Ok!

File: docs/UserGuide.md (hunk 2 above)
Comment (YEOWEIHNGWHYELAB) [reply docs/UserGuide.md]: Ok!

**Review by EdisonZhong17 — APPROVED** [review]
LGTM

**Review by jyxhazcake — APPROVED** [review]
LGTM

**Review by ZenoNexus — APPROVED** [review]
lgtm

**Pull request merged**